provisioning/miniprov/hare_mp/dhall/gencdf.dhall

```diff
@@ -26,6 +26,11 @@ let M0ClientDesc =
       , instances : Natural
       }
 
+let ClientPort =
```

Comment:
How about having something similar for m0server, name and port, instead of just port? We can then use names to map to the service types in Hare.

Reply:
@SwapnilGaonkar7 is this comment take care?

Reply:
Done